spec-main/api-web-contents-spec.ts

```diff
@@ -1245,6 +1245,35 @@ describe('webContents module', () => {
     });
   });
 
+  describe('crash behavior', () => {
+    let w: BrowserWindow;
+    beforeEach(async () => {
+      w = new BrowserWindow({ show: false, webPreferences: { nodeIntegration: true } });
```

Comment:
is `nodeIntegration` relevant here?

Perhaps it's worth also testing `sandbox`?